qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/commons/ResolvedJavaTypeInfo.java

```diff
@@ -145,7 +147,7 @@ private synchronized boolean computeIsIterable() {
 		if (iterableOf != null) {
 			return true;
 		}
-		boolean iterable = getSignature().equals("java.lang.Iterable");
+		boolean iterable = getName().equals("java.lang.Iterable");

```

Comment:
We could use/define a utility class containing Java type constants

Reply:
fixed